packages/zui-player/helpers/test-app.ts

```diff
@@ -45,6 +45,11 @@ export default class TestApp {
     if (bin) launchOpts.executablePath = bin;
     this.zui = await electron.launch(launchOpts);
 
+    // Pipe the stdout from the electron process into the test runner process
+    if (process.env['VERBOSE']) {
+      this.zui.process().stdout.pipe(process.stdout);
+    }
+
```

Comment:
For this one, if I run my tests via `VERBOSE=1 yarn e2e`, this change caused the `console.log()` messages I'd added in `packages/zed-node/src/lake.ts` to also show up in the shell window from which I ran `yarn e2e`.